src/FileUploadControl.php

```diff
@@ -151,10 +156,14 @@ public function __construct(string $name, int $maxFiles, ?string $maxFileSize =
 	/**
 	 * @static
 	 * @param Container     $systemContainer
-	 * @param array<mixed>  $configuration
+	 * @param array<mixed>|stdClass $configuration
 	 */
-	public static function register(Container $systemContainer, array $configuration = []): void
+	public static function register(Container $systemContainer, $configuration): void
 	{
+		if (is_object($configuration)) {
```

Comment:
K cemu slouzi tohle?

Reply:
Když se tam proměnná $configuration předá jako objekt, tak tento if ji převede na pole, které se používá dále v této metodě jako tam bylo dříve. Když jsem to testoval, tak to Nette předávalo jako objekt kvůli https://github.com/nette/forms/blob/4e64e56cbf2f26aa63535f3603c793ff73cc6c83/src/Forms/Container.php#L494.